### DISCUSSION TITLE
Using Reclaim/Cpanel to manage site

### DISCUSSION THREAD

**Discussion by waleskadelsol** [CollectionBuilder/collectionbuilder.github.io#107] [Q&A]
Hello! I have a question about site maintenance once it is already uploaded to cpanel. Today, I uploaded a new image and an updated CSV metadata file to my site via cpanel, but it did not update the site. Nothing happened to it -- just no updates. Does anyone know if there is a way to make changes/updates to one's CollectionBuilder site once it is already uploaded to cpanel? Or, do I have to deploy my collection with the changes and then re-upload it to cpanel/reclaim? Happy to elaborate more.

**Comment by evanwill** [comment]
@waleskadelsol yes, you will have to do `rake deploy` to rebuild the site on your local machine, then upload the files output in "_site" to your server again--replacing the old versions you deployed earlier. The build process is necessary to create all the pages and files with the updated information (i.e. adding a new item)--updating the CSV alone in your project does not update the collection without rebuilding and redeploying it! The process is the same as the first time you did it, replacing all the files where they currently are on your server.

Let us know if that helps!
